flutter/news/lib/features/frontpage/data/datasource/articles_data_source.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:news/features/frontpage/data/models/article_model.dart';
+
+abstract class ArticlesRemoteDataSource {
+  Future<List<ArticleModel>> getTopHeadlines();
```

Comment:
how many headlines do we expect after calling this?

Reply:
depends on the query used (we are forced to at least indicate a country like: "us" for example),  but around 30/40 articles but i think it can be more or less, will most likely limit it to 10 on an initial request